pkg/controller/controllerutil/controllerutil.go

```diff
@@ -91,6 +91,29 @@ func SetControllerReference(owner, controlled metav1.Object, scheme *runtime.Sch
 	return nil
 }
 
+// RemoveControllerReference is a helper method to make sure the given object removes an controller reference to the object provided.
```

Comment:
```suggestion
// RemoveControllerReference is a helper method to make sure the given object removes a controller reference to the object provided.
```

Reply:
Resolving because I updated this on the latest push.